src/fontcache.cpp

```diff
@@ -162,16 +221,20 @@ static bool IsDefaultFont(const FontCacheSubSetting &setting)
 uint GetFontCacheFontSize(FontSize fs)
 {
 	const FontCacheSubSetting &setting = *GetFontCacheSubSetting(fs);
-	return IsDefaultFont(setting) ? FontCache::GetDefaultFontHeight(fs) : setting.size;
+
+	if (IsDefaultFont(setting) && setting.size == 0) {
+		return FontCache::GetDefaultFontHeight(fs);
+	} else  {
+		return setting.size;
+	}
```

Comment:
An early return means you don't need an `else`.

```suggestion
	if (IsDefaultFont(setting) && setting.size == 0) return FontCache::GetDefaultFontHeight(fs);
	return setting.size;
```

Reply:
I'm aware. I did it for readability.  I'll change it.